vnext/src/babel-plugin-codegen/index.js

```diff
@@ -144,8 +144,11 @@ module.exports = function({parse, types: t}) {
             const viewConfig = generateViewConfig(this.filename, this.code);
             this.defaultExport.replaceWithMultiple(
               // [Win adding filename param see: https://github.com/facebook/react-native/pull/29230
-              parse(viewConfig, {filename: this.filename}).program.body,
-              // Win]
+              parse(viewConfig, {
```

Comment:
@acoates-ms/@adamfoxmann : Just FYI, as this seems to disable babelrc. Not sure if it affects what you are working on.